test/index.test.ts

```diff
@@ -23,7 +23,21 @@ async function endToEndTest(url: string) {
     const chains = await node.chains()
     expect(chains).not.toHaveLength(0)
 
-    const httpClient = new HttpChainClient(chains[0])
+    // currently the drand-client does not support the new G1/G2 swapped scheme
+    let chainToUse
+    for (const chain of chains) {
+        const info = await chain.info()
+        if (info.schemeID === 'pedersen-bls-chained' || info.schemeID === 'pedersen-bls-unchained') {
```

Comment:
What if it's "bls-unchained-on-g1" ??

Reply:
We don't test it because we don't support it. This test started breaking because it previously assumed `chains[0]` was one that it could process. Since we added `testnet-g`, it is now `chain[0]` and blows up.
Once we merge a fix for the g1 swap, we can revert this (and just test against all the networks)